docs/notebook-demos/map-algebra.ipynb

```diff
@@ -0,0 +1,655 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Polygonal Summary Methods\n",
+    "\n",
+    "In addition to local and focal operations, polygonal summaries can also be performed on `TiledRasterLayer`s. These are operations that are executed in the areas that intersect a given geometry and the layer.\n",
+    "\n",
+    "**Note**: It is important the given geometry is in the same projection as the layer. If they are not, then either incorrect and/or only partial results will be returned."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "tiled_layer.layer_metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Polygonal Min"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "poly_min = box(0.0, 0.0, 1.0, 1.0)\n",
+    "tiled_layer.polygonal_min(geometry=poly_min, data_type=int)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Polygonal Max"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "poly_max = box(1.0, 0.0, 2.0, 2.5)\n",
+    "tiled_layer.polygonal_min(geometry=poly_max, data_type=int)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Polygonal Sum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "poly_sum = box(0.0, 0.0, 1.0, 1.0)\n",
+    "tiled_layer.polygonal_min(geometry=poly_sum, data_type=int)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Polygonal Mean"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "poly_max = box(1.0, 0.0, 2.0, 2.0)\n",
+    "tiled_layer.polygonal_min(geometry=poly_max, data_type=int)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Cost Distance\n",
+    "\n",
+    "`cost_distance` is a function that will compute the accumulative cost to the nearest geometry of each cell in a friction layer. Cells that fall within the given geometry will have an accumulative cost of 0.0, while cells that contain noData values will continue to be represented with noData. If desired, a `max_distance` can also be set. This means that if a cell has a cost that goes past this max then noData will be set as the cell's value in the resulting layer.\n",
```

Comment:
That does read better, I'll replace what I had with that. I'm not really sure how the algorithm would behave in that instance, as I'm not familiar with how it's implemented in GeoTrellis.